nilearn_sandbox/datasets.py

```diff
@@ -0,0 +1,70 @@
+import os
+import cookielib
+import getpass
+
+from sklearn.datasets.base import Bunch
+
+from nilearn import datasets
+from nilearn._utils.compat import _urllib
+
+from ._utils.compat import _input
+
+
+def _get_nitrc_auth_cookie(login=None):
+    if login is None:
+        login = _input('login: ')
+    passwd = getpass.getpass('password: ')
```

Comment:
That's precisely what I want don't want to do. Block the possibility of writing the password in a script. I could move it up at the upper function level but since I don't see a reason to do it.